Desktop/data/importers/excelimporter.cpp

```diff
@@ -0,0 +1,116 @@
+//
+// Copyright (C) 2013-2024 University of Amsterdam
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 2 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with this program.  If not, see <http://www.gnu.org/licenses/>.
+//
+
+#include "excelimporter.h"
+#include "data/importers/excel/excel.h"
+#include "data/importers/excel/excelimportcolumn.h"
+#include <string>
+#include <QFileInfo>
+#include <QDebug>
+
+using namespace std;
```

Comment:
Im not a big fan of `using namespace std;` we still have it in some older code.
But adding `std::` in front of functions is not so bad and it avoids possible misdirections for functions like `max()` or `min()` and other, perhaps unexpected, collisions.

And I see that `std::` is being used anyway so it shouldnt be a problem to remove it anyway.